libs/tup-components/src/system-status/details/SystemNavBar.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { QueryNavItem } from '@tacc/core-wrappers';
+import { useSystemMonitor } from '@tacc/tup-hooks';
+import { InlineMessage, LoadingSpinner } from '@tacc/core-components';
+import { SystemDetailProps } from '..';
+
+export const SystemNavBar: React.FC<SystemDetailProps> = ({
+  // To set default prop for system if none selected
+  tas_name = 'frontera',
+}) => {
+  const { data, error, isLoading } = useSystemMonitor();
+
+  if (error)
+    return (
+      <InlineMessage type="warning">
+        Unable to retrieve navigation bar.
+      </InlineMessage>
+    );
+  if (isLoading) return <LoadingSpinner />;
+
+  return (
+    data &&
+    data.map((system) => (
+      <QueryNavItem
```

Comment:
It doesn't work exactly right when using NavBar. When using NavBar, if a user selects System Status from the main nav bar, it goes to /system-status and it doesn't activate the Frontera link. This solution was to prevent no system being active in the SystemNavBar by passing in the default prop for tas_name in if one was not present URL.